server/plugin/command.go

```diff
@@ -501,6 +518,163 @@ func (p *Plugin) handleIssue(_ *plugin.Context, args *model.CommandArgs, paramet
+		siteURL := *p.API.GetConfig().ServiceSettings.SiteURL + Manifest.Props["inbound_webhook_url"].(string)
+		var hook github.Hook
+		var config = make(map[string]interface{})
+		config["secret"] = p.getConfiguration().WebhookSecret
+		if len(parameters) >= 1 {
+			parameters = parameters[1:]
+			for _, val := range parameters {
+				switch {
+				case strings.Contains(val, "https://"):
+					config["url"] = val
+				case strings.Contains(val, "true") || strings.Contains(val, "false"):
+					insecureVal, _ := strconv.ParseBool(val)
+					config["insecure_ssl"] = insecureVal
+				case strings.Contains(val, "application/json") || strings.Contains(val, "x-www-form-urlencoded"):
+					config["content_type"] = val
+				default:
+					if strings.Contains(val, "*") && len(val) == 1 {
+						hook.Events = []string{"*"}
+						break
+					}
+					if err := p.CheckOptionsValid(strings.Split(val, ",")); err != nil {
+						return err.Error()
+					}
+					hook.Events = strings.Split(val, ",")
+				}
+			}
+		}
+		if _, exists := config["url"]; !exists {
+			config["url"] = siteURL
+		}
+		if _, exists := config["insecure_ssl"]; !exists {
+			config["insecure_ssl"] = false
+		}
+		if _, exists := config["content_type"]; !exists {
+			config["content_type"] = "x-www-form-urlencoded"
+		}
+		if len(hook.Events) == 0 {
+			hook.Events = []string{"*"}
+		}
+		hook.Config = config
+		githubHook, _, err := p.CreateHook(ctx, githubClient, owner, repo, hook)
+		if err != nil {
+			return err.Error()
+		}
+
+		hookDetails, _, err := p.GetHook(ctx, githubClient, owner, repo, *githubHook.ID)
+		if err != nil {
+			return err.Error()
+		}
+
+		hookURL := baseURL + owner
+		if repo != "" {
+			hookURL += "/" + repo
+		}
+		hookURL += "/settings/hooks/"
+		txt := "Webhook Created Successfully \n"
+		hookID := strconv.Itoa(int(*githubHook.ID))
+		txt += fmt.Sprintf(" *  [%s](%s) :  -  %s \n", hookID, hookURL+hookID, strings.Join(hookDetails.Events, " , "))
+		return txt
+	case list:
+
+		if len(parameters) != 1 {
+			return "Unknown action, please use `/github help` to see all actions available."
+		}
+
+		owner, repo := parseOwnerAndRepo(parameters[0], p.getBaseURL())
+		var githubHooks []*github.Hook
+		var githubResponse *github.Response
+		var err error
+		opt := &github.ListOptions{
+			PerPage: 10,
+		}
+		if repo == "" {
+			for {
+				githubHooks, githubResponse, err = githubClient.Organizations.ListHooks(ctx, owner, opt)
+				if err != nil {
+					return err.Error()
+				}
+				if githubResponse.NextPage == 0 {
+					break
+				}
+				opt.Page = githubResponse.NextPage
+			}
+		} else {
+			for {
+				githubHooks, githubResponse, err = githubClient.Repositories.ListHooks(ctx, owner, repo, opt)
+				if err != nil {
+					return err.Error()
+				}
+				if githubResponse.NextPage == 0 {
+					break
+				}
+				opt.Page = githubResponse.NextPage
+			}
+		}
+
+		var txt string
+		if len(githubHooks) == 0 {
+			txt = "Currently there are no webhook in this [Owner/Repo]"
```

Comment:
Can we be more specific here? Like:

```suggestion
			txt = "There are currently no GitHub webhooks created for the [%s](https://github.com/%s) org or the [%s/%s](https://github.com/%s/%s) repository."
```

Which should ultimately render as:

There are currently no GitHub webhooks created for the [mattermost](https://github.com/mattermost) org or the [mattermost/mattermost-plugin-github](https://github.com/mattermost/mattermost-plugin-github) repository.